test/gui/drone/notification_template.sh

```diff
@@ -8,11 +8,13 @@ COMMIT_SHA_SHORT=${DRONE_COMMIT:0:8}
 
 GUI_LOG="${CACHE_ENDPOINT}/${CACHE_BUCKET}/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/guiReportUpload/index.html"
 SERVER_LOG="${CACHE_ENDPOINT}/${CACHE_BUCKET}/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/guiReportUpload/serverlog.log"
+STACKTRACE="${CACHE_ENDPOINT}/${CACHE_BUCKET}/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/guiReportUpload/stacktrace"
 
 CURL="curl --write-out "%{http_code}" --silent --output /dev/null"
 
 GUI_STATUS_CODE=$($CURL "$GUI_LOG")
 SERVER_STATUS_CODE=$($CURL "$SERVER_LOG")
+STACKTRACE_STATUS_CODE=$($CURL "$STACKTRACE")
```

Comment:
The escaping here is incomplete. Please consider running `shellcheck` on this script (and possibly others, too).